publications/abstracts/reservoir-computing/methodology.tex

```diff
@@ -0,0 +1,76 @@
+\section{Methodology}
+
+This work consists of three parts, data generation model training, and
+prediction. To generate
+data needed for training our RC model we used the \texttt{TrainARMA}
+functionality
+in the \texttt{RAVEN} tool from INL to generate synthetic histories. These
+scenarios
+are passed to the RC model for training and testing.
+
+\subsection{Training Data Generation}
+In this work we use the scenario generation method described in Baker et. al
+using the \texttt{RAVEN} framework \cite{baker_optimal_2018}. The algorithm for
+producing synthetic histories is:
+\begin{enumerate}
+	\item Create a ``typical year`` from historical data.
+	\item Fit an Autoregressive Moving Average (ARMA) model using
+	\texttt{TrainARMA}.
+	\item Use Monte Carlo sampling to generate synthetic histories from ARMA
+	 model.
+\end{enumerate}
+
+We perform this process for three datasets: Electricity demand, wind generation,
+and solar generation. This allows us to train an RC model to predict generation
+from wind and solar farms and to predict the net demand profile given by
+\begin{equation}
+	\label{eqn:net-demand}
+	\begin{split}
+		D_{net} & = (D_{total} - P_{wind} - P_{pv})_h \text{ $\forall$ $h$ in }
+		[0,8759]
+	\end{split}
+\end{equation}
+
+Where $D_{total}$ is the total demand at a given hour, $h$, of a synthetic
+demand profile from the
+ARMA model.
+
+\subsection{Reservoir Computing Model}
+
+Unlike a typical feed forward neural network, a reservoir computer (sometimes
```

Comment:
I think that, in order to reference a "feed forward neural network" as if it's typical, you'll need to introduce it to the reader.